### PR TITLE
Revert VRF/VDF

### PR DESCRIPTION
**Pull request by harmony-ek**
Revert VRF/VDF for the s3 release branch.

### CODE REVIEW

File: api/service/syncing/syncing.go [hunk 1]
```diff
@@ -317,7 +318,7 @@ func (sc *SyncConfig) GetBlockHashesConsensusAndCleanUp() bool {
 		Int("maxFirstID", maxFirstID).
 		Int("maxCount", maxCount).
 		Msg("[SYNC] block consensus hashes")
-	if float64(maxCount) >= core.ShardingSchedule.ConsensusRatio()*float64(len(sc.peers)) {
+	if float64(maxCount) >= ConsensusRatio*float64(len(sc.peers)) {
```
Comment (LeoHChen) [inline api/service/syncing/syncing.go]: is this change needed?

File: internal/configs/sharding/localnet.go [hunk 2]
```diff
@@ -71,21 +65,6 @@ func (ls localnetSchedule) IsLastBlock(blockNum uint64) bool {
 	}
 }
 
-func (ls localnetSchedule) VdfDifficulty() int {
-	return localnetVdfDifficulty
-}
-
-// ConsensusRatio ratio of new nodes vs consensus total nodes
-func (ls localnetSchedule) ConsensusRatio() float64 {
```
Comment (LeoHChen) [inline internal/configs/sharding/localnet.go]: this shouldn't be reverted

File: internal/configs/sharding/mainnet.go [hunk 3]
```diff
@@ -84,21 +78,6 @@ func (ms mainnetSchedule) IsLastBlock(blockNum uint64) bool {
 	}
 }
 
-func (ms mainnetSchedule) VdfDifficulty() int {
-	return mainnetVdfDifficulty
-}
-
-// ConsensusRatio ratio of new nodes vs consensus total nodes
-func (ms mainnetSchedule) ConsensusRatio() float64 {
```
Comment (LeoHChen) [inline internal/configs/sharding/mainnet.go]: same here?

File: api/service/syncing/syncing.go (hunk 1 above)
Comment (harmony-ek) [reply api/service/syncing/syncing.go]: Right, reverted.  This was for lowering block syncing quorum for localnet test and had nothing to do with VRF/VDF.
